symphony/content/content.blueprintsdatasources.php

```diff
@@ -331,7 +331,7 @@ public function __form()
         $fieldset = new XMLElement('fieldset');
         $this->setContext($fieldset, array('sections', 'system'));
         $fieldset->appendChild(new XMLElement('legend', __('Error Conditions')));
-        $p = new XMLElement('p', __('Enabling these conditions will send a <code>404 page not found</code> error.'));
+        $p = new XMLElement('p', __('If enabled conditions are met, a <code>"404 page not found"</code> error will be sent.'));
         $p->setAttribute('class', 'help');
```

Comment:
1. > Do we need the quotes if the response is marked up as code?

No, but on low constrast screens, it made the error name pop out.

2. I also like @nilshoerrmann proposition. The "if" part is not necessary.

Also thanks @michael-e for pointing out the fact the the error name does not include the word 'page'. I will never get enough of HTTP status code geekness.